src/AnnotationThread.js

```diff
@@ -585,7 +585,8 @@ class AnnotationThread extends EventEmitter {
                 threadID: this.threadID
             },
             message,
-            createdBy: this.api.user
+            createdBy: this.api.user,
```

Comment:
Is there a `getUser()` function on the api instance? Just to be safe

Reply:
There isn't right now but I can add one in a separate PR.